### DISCUSSION TITLE
Aggregate Disk Utilization Report

### DISCUSSION THREAD

**Discussion by KlausHub** [NetApp/harvest#777] [Q&A] [answered]
Hi there,

in  old **Harvest 1.0 - Node Dashboard** we had a view on the **Aggregate Disk Utilization**, which we miss in Harvest 2.0.
It was very useful for Performance debugging to help to see, which  of our Disk-Based Aggregates may be the bottleneck.

![image](https://user-images.githubusercontent.com/97510878/148913605-6fc5b7e0-840d-486f-b7de-71ec5ab86d6a.png)

Can you please implement that it also on the Harvest 2.0 or guide me there to find it (if already existing).

Best Regards,
Klaus

**Comment by cgrinds** [comment, marked as answer]
Hi @KlausHub does the `Disk Utilization` panel on the Disk dashboard help? That panel includes the breakdown and is tied to the node filter at the top of the page.

<img width="503" alt="image" src="https://user-images.githubusercontent.com/242252/148950156-efa665fc-5ccd-40ac-be51-7e4dad166f3a.png">

**Reply by KlausHub** [reply]
HI @cgrinds,
Thank you very much, that was exactly i was searching for in Harvest 2.0.
